src/components/item_page/ItemPage.js

```diff
@@ -6,9 +6,11 @@ import { convertPenniesToDollars } from 'utilities/price';
 import ItemBidForm from 'components/item_bids/ItemBidForm';
 import Error from 'components/UI/Error';
 
-export const ItemPage = ({ item, error, children, handleBidSubmit }) => {
+export const ItemPage = ({ item, error, children, handleBidSubmit, success }) => {
   const { title, description, price, end_date, image, current_highest_bid } = item;
-  const minimumPrice = price === current_highest_bid ? price : current_highest_bid + 100;
+  const currentPrice = current_highest_bid ? current_highest_bid : price;
```

Comment:
You might be able to replace this with `currentPrice = current_highest_bid || price` unless you want it to be possible for `currentPrice` to be 0 (I would imagine that you'd never show a 0 bid though)

Reply:
Updating!